local-compose.sh

```diff
@@ -6,16 +6,11 @@
 #
 #
 TOP=$(git rev-parse --show-toplevel)
-BACKEND=${TOP}/opt/backend
+BACKEND=${TOP}/backend
 
-CPT_BACKEND_PORT=${CPT_BACKEND_PORT:-8000}
-CPT_FRONTEND_PORT=${CPT_FRONTEND_PORT:-3000}
-CPT_CONFIG=${CPT_CONFIG:-"${BACKEND}/ocpperf.toml"}
-podman rm -f front back
+podman compose --file ${TOP}/compose.yaml down
```

Comment:
Doesn't this generate an error if the "composition" isn't up? Unfortunately, it seems that even `podman compose ps` *fails* if nothing's running, which seems absurd. Maybe a good argument to move away from `compose`, however we choose to do that.

In any case, given that we can't apparently be "smart" about this, I think it might be better just to remove this. The old version failed if the ports were already in use--this can, too.

Reply:
I am not seeing an error

```shell
❯ podman compose down
[+] Running 5/5
 ✔ Container frontend             Removed                                         0.2s 
 ✔ Container db-seed              Removed                                         0.0s 
 ✔ Container backend              Removed                                        10.2s 
 ✔ Container searchdb-ocp         Removed                                         0.4s 
 ✔ Network cpt-dashboard_e2e-pod  Removed                                         0.0s 
❯ echo $?
0
❯ podman compose down
❯ echo $?
0
```

Reply:
do you get that error when you provide it the filepath to `--file`?